src/components/BaseModal/BaseModal.tsx

```diff
@@ -0,0 +1,43 @@
+import React, { useEffect } from 'react';
+import './BaseModal.css';
+import BaseCard from '../BaseCard';
+
+interface BaseModalProps {
+  isOpen?: boolean;
+  renderHeader?: () => React.ReactNode;
+  onClose?: (value: boolean) => void;
+  children?: React.ReactNode;
+  className?: string;
+}
+
+function BaseModal({ isOpen = false, renderHeader, onClose, children, className = '' }: BaseModalProps) {
+  useEffect(() => {
+    if (isOpen) {
+      document.body.style.overflow = 'hidden';
+    } else {
+      document.body.style.overflow = 'auto';
+    }
+  }, [isOpen]);
```

Comment:
Hacer un espacio entre lineas para mejorar legibilidad
```suggestion
  }, [isOpen]);
  
```